File: updraft_sdk/src/main/java/com/apsswithlove/updraft_sdk/api/request/FeedbackMobileRequest.java
package com.apsswithlove.updraft_sdk.api.request;

import com.google.gson.annotations.SerializedName;

public class FeedbackMobileRequest {

    @SerializedName("sdk_key")
    private String mSdkKey;

    @SerializedName("app_key")
    private String mAppKey;

    @SerializedName("image")
    private String mImage;

    @SerializedName("tag")
    private String mTag;

    @SerializedName("description")
    private String mDescription;

    @SerializedName("email")
    private String mEmail;

    @SerializedName("build_version")
    private String mBuildVersion;

    @SerializedName("system_version")
    private String mSystemVersion;

    @SerializedName("device_name")
    private String mDeviceName;

    @SerializedName("device_uuid")
    private String mDeviceUuid;

    @SerializedName("navigation_stack")
    private String mNavigationStack;

    public String getSdkKey() {
        return mSdkKey;
    }

    public void setSdkKey(String sdkKey) {
        mSdkKey = sdkKey;
    }

    public String getAppKey() {
        return mAppKey;
    }

    public void setAppKey(String appKey) {
        mAppKey = appKey;
    }

    public String getImage() {
        return mImage;
    }

    public void setImage(String image) {
        mImage = image;
    }

    public String getTag() {
        return mTag;
    }

    public void setTag(String tag) {
        mTag = tag;
    }

    public String getDescription() {
        return mDescription;
    }

    public void setDescription(String description) {
        mDescription = description;
    }

    public String getEmail() {
        return mEmail;
    }

    public void setEmail(String email) {
        mEmail = email;
    }

    public String getBuildVersion() {
        return mBuildVersion;
    }

    public void setBuildVersion(String buildVersion) {
        mBuildVersion = buildVersion;
    }

    public String getSystemVersion() {
        return mSystemVersion;
    }

    public void setSystemVersion(String systemVersion) {
        mSystemVersion = systemVersion;
    }

    public String getDeviceName() {
        return mDeviceName;
    }

    public void setDeviceName(String deviceName) {
        mDeviceName = deviceName;
    }

    public String getDeviceUuid() {
        return mDeviceUuid;
    }

    public void setDeviceUuid(String deviceUuid) {
        mDeviceUuid = deviceUuid;
    }

    public String getNavigationStack() {
        return mNavigationStack;
    }

    public void setNavigationStack(String navigationStack) {
        mNavigationStack = navigationStack;
    }
}

File: updraft-core/src/commonMain/kotlin/com/appswithlove/updraft/api/request/FeedbackMobileRequest.kt
package com.appswithlove.updraft.api.request

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class FeedbackMobileRequest(
    @SerialName("sdk_key") val sdkKey: String? = null,
    @SerialName("app_key") val appKey: String? = null,
    @SerialName("image") val image: String? = null,
    @SerialName("tag") val tag: String? = null,
    @SerialName("description") val description: String? = null,
    @SerialName("email") val email: String? = null,
    @SerialName("build_version") val buildVersion: String? = null,
    @SerialName("system_version") val systemVersion: String? = null,
    @SerialName("device_name") val deviceName: String? = null,
    @SerialName("device_uuid") val deviceUuid: String? = null,
    @SerialName("navigation_stack") val navigationStack: String? = null,
)
